### PR TITLE
Enhancement: Periodically check and include dhcp server status in init data #54

### PR DESCRIPTION
**Pull request by blackfirespace**

### CODE REVIEW

File: net2/HostManager.js [hunk 1]
```diff
@@ -975,6 +975,7 @@ module.exports = class HostManager {
         if (json.mode === "dhcp") {
           await this.dhcpRangeForInit("alternative", json);
           await this.dhcpRangeForInit("secondary", json);
+          json.dhcpserverstatus = await rclient.getAsync("sys:scan:dhcpserver");
```
Comment (drahnieR) [inline net2/HostManager.js]: use camelCase for variable names

File: extension/dhcp/dhcp.js [hunk 2]
```diff
@@ -89,6 +89,42 @@ async function dhcpDiscover(intf) {
...
 }
 
+async function dhcpServerStatus(serverIp) {
+  let result = false;
+  let cmd = util.format('sudo nmap -sU -p 67 --script=dhcp-discover %s -oX - | %s', serverIp, xml2jsonBinary);
+  log.info("Running command:", cmd);
+  try {
+    const cmdresult = await execAsync(cmd);
+    let output = JSON.parse(cmdresult.stdout);
+    let kvs = output.nmaprun && output.nmaprun.host
```
Comment (drahnieR) [inline extension/dhcp/dhcp.js]: consider using lodash.get() for readability
https://lodash.com/docs/4.17.15#get

File: extension/dhcp/dhcp.js [hunk 3]
```diff
@@ -89,6 +89,42 @@ async function dhcpDiscover(intf) {
...
+
 module.exports = {
-  dhcpDiscover: dhcpDiscover
-}
\ No newline at end of file
+  dhcpDiscover: dhcpDiscover,
+  dhcpServerStatus: dhcpServerStatus,
+}
+
+/*
```
[the remark this answers is not in the document]
Comment (drahnieR) [reply extension/dhcp/dhcp.js]: why this is resolved?

If we need this for test, write a script, or add something to `test/`

[the remark this answers is not in the document]
Comment (blackfirespace) [reply extension/dhcp/dhcp.js]: There was this code when merging code before, because it was written by someone else, so it was not deleted.

**Pull request merged**